repo-agent/repowatch/controllers/repowatch_controller.go

```diff
@@ -481,42 +507,23 @@ func (r *RepoWatchReconciler) reconcileReviewSandboxes(ctx context.Context, repo
 			if err := r.Delete(ctx, &sandbox); err != nil {
 				log.Error(err, "unable to delete sandbox", "sandbox", sandbox.GetName())
 			}
-			totalSandboxes--
```

Comment:
should we have this ?

Reply:
No. It is a mistake. I have reintroduced this code, and added unit tests to verify it works.